galaxy_ng/app/api/ui/urls.py

```diff
@@ -64,6 +64,19 @@
         "namespaces/",
         viewsets.ContainerNamespaceViewSet.as_view({'get': 'list'}),
         name='container-namespace-list'),
+    path(
+        "registries/<str:pk>/",
+        viewsets.SyncConfigContainerViewSet.as_view({'get': 'retrieve', 'put': 'update'}),
+        name='execution-environments-registries-detail'),
```

Comment:
I _think_ the convention here is to have this singular, so `registry-detail` and `registry-list`